lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenProjectCache.java

```diff
@@ -210,7 +200,7 @@ public InputStream getInputStream() throws IOException {
 
 	private ProjectBuildingRequest newProjectBuildingRequest() {
 		ProjectBuildingRequest request = new DefaultProjectBuildingRequest();
-		request.setSystemProperties(systemProperties);
+		request.setSystemProperties(lemminxMavenPlugin.getSystemProperties());
```

Comment:
It would be better to use `mavenRequest.getSystemProperty()` here, so we only have to care about a single place.

Reply:
ACK